MinecraftClient/Commands/Move.cs

```diff
@@ -1,38 +1,36 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Text;
 using MinecraftClient.Mapping;
 
 namespace MinecraftClient.Commands
 {
     public class Move : Command
     {
         public override string CmdName { get { return "move"; } }
-        public override string CmdUsage { get { return "move <on|off|get|up|down|east|west|north|south|x y z>"; } }
-        public override string CmdDesc { get { return "walk or start walking."; } }
+        public override string CmdUsage { get { return "move <on|off|get|up (-f)|down (-f)|east (-f)|west (-f)|north (-f)|south (-f)|x y z>"; } }
```

Comment:
I think the help message here became too complex and hard to read. 

How about:
```suggestion
        public override string CmdUsage { get { return "move <on|off|get|up|down|east|west|north|south|x y z> [-f]"; } }
```